core/lib/generators/spree/custom_user/custom_user_generator.rb

```diff
@@ -22,10 +22,12 @@ def generate
       migration_template 'migration.rb.tt', "db/migrate/add_spree_fields_to_custom_user_table.rb"
       template 'authentication_helpers.rb.tt', "lib/spree/authentication_helpers.rb"
 
-      file_action = File.exist?('config/initializers/spree.rb') ? :append_file : :create_file
-      send(file_action, 'config/initializers/spree.rb') do
-        "Rails.application.config.to_prepare do\n  require_dependency 'spree/authentication_helpers'\nend\n"
-      end
+      initializer 'solidus_authentication', <<~RUBY
+        Rails.application.config.to_prepare do
+          ApplicationController.include Spree::AuthenticationHelpers, Spree::CurrentUserHelpers
```

Comment:
I might be missing something, but why do we need to include those helpers in the user's main controller?

Reply:
Short (incomplete) answer: we're just keeping the previous behavior

Longer version: We could probably include them in `Spree::BaseController` but the currently released `solidus_auth_devise` has the default `ApplicationController` set to be the `Devise.parent_controller` used for all devise controller. So I'm pretty sure including in BaseController would be a problem, although I didn't try.

Reply:
I see. Thanks. Something for another PR.